android-pay/src/test/java/com/stripe/wrap/pay/utils/CartManagerTest.java

```diff
@@ -36,6 +36,12 @@
 @Config(sdk = 23)
 public class CartManagerTest {
 
+    @Test(expected = RuntimeException.class)
```

Comment:
The no-args constructor testing the no-args runtime exception for AndroidPayConfiguration can't reliably be tested because AndroidPayConfiguration is a singleton, so it depends on the run order of tests, which I can't guarantee in builds.